extension/test/e2e/klarna.spec.js

```diff
@@ -115,18 +131,53 @@ describe('::klarnaPayment::', () => {
           redirectPaymentResult
         )
 
-        // Capture the payment
-        // paymentAfterCapture = await capturePayment({
-        //   payment: paymentAfterHandleRedirect,
-        // })
-        // logger.debug(
-        //   'klarna::paymentAfterCapture:',
-        //   JSON.stringify(paymentAfterCapture)
-        // )
+        const notificationInteraction = await waitUntil(
+          async () =>
+            await fetchNotificationInterfaceInteraction(
+              ctpClient,
+              paymentAfterCreateSession.id
+            )
+        )
+
+        if (notificationInteraction) {
+          // #3 - Capture the payment
+          paymentAfterCapture = await capturePayment({
+            payment: paymentAfterCreateSession,
+          })
+        }
+
+        const captureNotificationInteraction = await waitUntil(
+          async () =>
+            await fetchNotificationInterfaceInteraction(
+              ctpClient,
+              paymentAfterCreateSession.id,
+              'capture'
+            )
+        )
+        const captureNotification =
+          captureNotificationInteraction.fields.notification
+        if (captureNotification) {
+          const captureNoticationRequestItem =
+            JSON.parse(captureNotification).NotificationRequestItem
+          captureEventCode = captureNoticationRequestItem.eventCode
+          capturePspReference = captureNoticationRequestItem.pspReference
+          captureOriginalPspReference =
+            captureNoticationRequestItem.originalReference
+          captureSuccess = captureNoticationRequestItem.success
```

Comment:
Good to see that you are already asserting the response of Notification.

Reply:
As discussed, it is still good to implement as long as notification works. Otherwise we check the issue of notification in next PR.